app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -47,6 +47,12 @@ def update
           selected_location_details: update_params.as_json,
           issuer: current_sp&.issuer,
         )
+
+        if document_capture_session
```

Comment:
should we consider making 1 database call instead of 2 database calls back to back?